404.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
new 404 page for bad links... i think this might work throughout the site. might be nice to add some more link structure to this page.